bookwyrm/templates/lists/list.html

```diff
@@ -40,6 +81,16 @@
                             <input type="hidden" name="item" value="{{ item.id }}">
                             <button type="submit" class="button is-small is-danger">{% trans "Remove" %}</button>
                         </form>
+                    </div>
+                    <div class="card-footer has-background-white-bis">
+                        <div>
+                            <form name="set-position" method="post" action="{% url 'list-set-book-position' item.id %}" class="card-footer-item">
+                                {% csrf_token %}
+                                <label for="input-list-position" class="is-sr-only">{% trans "Set list position" %}</label>
+                                <input id="input-list-position" class="input" type="text" name="position" value="">
+                                <button aria-label="{% trans 'Click to set chosen position' %}" type="submit" class="button is-small is-info">{% trans "Set list position" %}</button>
```

Comment:
I don't think the button needs an `aria-label` field, since the button text will be read by the screen reader.

Reply:
Took that out.